Warning: application@android:isGame was tagged to replace other declarations but no other declaration present

### Godot version

3.4.beta 009aa63a5723eec59256a2bcd8ee2a038d1aa958

### System information

Ubuntu 21.04

### Issue description

When building an Android AAB there is a new warning:
```
AndroidManifest.xml:16:5-27:19 Warning: application@android:isGame was tagged at AndroidManifest.xml:16 to replace other declarations but no other declaration present
```
**Note:** This does not happen when building an APK.

Regression from #50028 cherry-picked to 3.4 with #50202 cd64bcdae56ff80736a12968b236beae9b0ab720

### Steps to reproduce

Install the Android Build Template. Create an Android preset. Select `Use Custom Build`. Select `Export Format`: `Export AAB`. Try to export the project.

### Minimal reproduction project

Any project will do, but here is a simple one:
[HelloAndroid.zip](https://github.com/godotengine/godot/files/6707817/HelloAndroid.zip)

I've just tried again with 7c8cc037ad86fe1963ce978d20200e875472fad4 i.e. with #50227, and now the Gradle build fails! :confused: 
```
FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':processDebugManifest'.
> com.android.manifmerger.ManifestMerger2$MergeFailureException: org.xml.sax.SAXParseException; lineNumber: 19; columnNumber: 7; The element type "manifest" must be terminated by the matching end-tag "</manifest>".

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org
```

--- a/platform/android/export/gradle_export_util.h
+++ b/platform/android/export/gradle_export_util.h
@@ -264,11 +264,11 @@ String _get_application_tag(const Ref<EditorExportPreset> &p_preset, bool p_has_
 	String manifest_application_text = vformat(
 			"    <application android:label=\"@string/godot_project_name_string\"\n"
 			"        android:allowBackup=\"%s\"\n"
-			"        android:icon=\"@mipmap/icon\"\n"
 			"        android:isGame=\"%s\"\n"
 			"        android:requestLegacyExternalStorage=\"%s\"\n"
 			"        tools:replace=\"android:allowBackup,android:isGame,android:requestLegacyExternalStorage\"\n"
-			"        tools:ignore=\"GoogleAppIndexingWarning\">\n\n"
+			"        tools:ignore=\"GoogleAppIndexingWarning\"\n"
+			"        android:icon=\"@mipmap/icon\" >\n\n"
 			"        <meta-data tools:node=\"remove\" android:name=\"xr_mode_metadata_name\" />\n",
 			bool_to_string(p_preset->get("user_data_backup/allow")),
 			bool_to_string(p_preset->get("package/classify_as_game")),

--- a/platform/android/java/app/AndroidManifest.xml
+++ b/platform/android/java/app/AndroidManifest.xml
@@ -28,7 +28,7 @@
     <!-- If you want to add tags manually, do before it. -->
     <!-- WARNING: This should stay on a single line until the parsing code is improved. See GH-32414. -->
     <!-- TODO: Remove the 'requestLegacyExternalStorage' attribute when https://github.com/godotengine/godot/issues/38913 is resolved -->
-    <application android:label="@string/godot_project_name_string" android:allowBackup="false" android:icon="@mipmap/icon" android:isGame="true" android:requestLegacyExternalStorage="false" tools:ignore="GoogleAppIndexingWarning" >
+    <application android:label="@string/godot_project_name_string" android:allowBackup="false" android:isGame="true" android:requestLegacyExternalStorage="false" tools:ignore="GoogleAppIndexingWarning" android:icon="@mipmap/icon" >
 
         <!-- Records the version of the Godot editor used for building -->
         <meta-data

